tests/test_freshdesk_pagination.py

```diff
@@ -1,58 +1,91 @@
-from tap_tester import menagerie, connections, runner
+from math import ceil
+from tap_tester import menagerie, connections, runner, LOGGER
 import re
```

Comment:
Can we remove unused import re and menagerie

Reply:
Removed